.ats/main.yaml

```
@@ -0,0 +1,10 @@
+app-tests-app-config-file: tests/test-values.yaml
+app-tests-deploy-namespace: monitoring
+
+smoke-tests-cluster-type: kind
+
+upgrade-tests-cluster-type: kind
+upgrade-tests-app-catalog-url: https://giantswarm.github.io/giantswarm-catalog
```

The tests are failing because this app is not in the default catalog, but in the control-plane catalog.
```suggestion
upgrade-tests-app-catalog-url: https://giantswarm.github.io/control-plane-catalog
```